src/styles/fonts/_roboto.scss

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Regular.woff2) format('woff2');
+  src: url(../fonts/Roboto-Regular.ttf) format('truetype');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Medium.woff2) format('woff2');
+  src: url(../fonts/Roboto-Medium.ttf) format('truetype');
```

Comment:
Similarly, the `src` property for the Roboto-Medium font should include both `woff2` and `ttf` formats in one declaration.